testing/hello_world_linux_init/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
```

Comment:
Personally I would move this crate up to the top level and call it `oak_hello_world_linux_init` or similar.

Reply:
Done.